department-of-veterans-affairs/caseflow
# Only show docketing action if FT is enabled for user

Resolves [CASEFLOW-2301](https://vajira.max.gov/browse/CASEFLOW-2301)

### Description
Only show docketing action when the `docket_vha_appeals` feature toggle is enabled

### Acceptance Criteria

### Testing Plan
1. As a BVA Intake user, visit an appeal that has been assigned by CAMO as ready for docketing
2. Ensure the action dropdown isn't shown.
3. Add the `docket_vha_appeals` and revisit the appeal. The action dropdown should show up, with the option to docket the appeal.

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1469" alt="Screen Shot 2021-11-19 at 1 22 45 PM" src="https://user-images.githubusercontent.com/640182/142672636-9f8677db-2b89-41b5-a4f9-790d0e6f2b29.png">|<img width="1464" alt="Screen Shot 2021-11-19 at 1 23 09 PM" src="https://user-images.githubusercontent.com/640182/142672659-899018fa-691b-41a7-836b-af2aef109110.png">
